### PR TITLE
Add simplified methods for sell and buy orders

### PR DESCRIPTION
**Pull request by kainanaina**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -29,5 +29,6 @@ module.exports = {
     "import/no-cycle": "error",
     "no-unused-vars": "off",
     "prefer-template": "error",
+    "crlf": "auto",
```
Comment (leeyikjiun) [inline .eslintrc.js:32]: Can't find the eslint rule for this. What will happen if this rule isn't here?

File: packages/sdk/src/types.ts [hunk 2]
```diff
@@ -37,6 +37,14 @@ export interface MakeOrderParams {
   expirationTime?: Date;
 }
 
+export interface MakeOrderParamsSimplified {
```
Comment (leeyikjiun) [inline packages/sdk/src/types.ts:40]:
```typescript
export interface MakeSellOrderParams {

export type MakeBuyOrderParams = MakeSellOrderParams
```

File: packages/sdk/src/index.ts [hunk 3]
```diff
@@ -99,6 +100,36 @@ export default class Gomu {
     );
   }
 
+  async makeSellOrder({
+    assets,
+    amount,
+    paymentTokenAddress,
```
Comment (leeyikjiun) [inline packages/sdk/src/index.ts:106]: Think we can squash these 2 params to just `erc20Asset`. Wanted to use `baseAssets` and `quoteAssets` to make it even more general, but @kennethtiong feels that it's not layman enough.

File: packages/sdk/src/index.ts [hunk 4]
```diff
@@ -99,6 +100,36 @@ export default class Gomu {
...
+    taker,
+    expirationTime
+  }: MakeOrderParamsSimplified) {
```
Comment (leeyikjiun) [inline packages/sdk/src/index.ts:109]: Add return type for function

**Review by leeyikjiun — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (kainanaina) [reply .eslintrc.js]: Removed for now, will see how to fix it on a git level on my side.

**Review by leeyikjiun — APPROVED** [review]

**Pull request merged**